### PR TITLE
[FIX] Split out tractography

### PR DESCRIPTION
**Pull request by oesteban**
This PR splits the tractography section from the diffusion derivatives
document, so that #5 is easier to merge.
The new ``05-diffusion-derivatives-tractography.md`` file will remain
orphaned, but kept there as a base for the time we tackle tractography.
It shouldn't be merged into the derivatives branch until it is ready.

### CODE REVIEW

File: src/05-derivatives/05-diffusion-derivatives-tractography.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+| GLOBAL | SpinGlass  | Fillard P. et al. (2009)                                                                                                             |
+| GLOBAL | ENS        | Takemura et al. (2016)                                                                                                               |
+| GLOBAL | Other      | Mangin, J.F. et al. (2013); Aganj, I. et al. (2011); Neher, P.F. et al. (2012); Jbabdi, S., et al. (2007); Reisert, M. et al. (2011) |
+
+The JSON sidecar contains the following key/value pairs:
+
+| **Key name**       | **Description**                                                                                                      |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------- |
+| TractographyClass  | REQUIRED. Allowed values: `local`, `global`                                                                          |
+| TractographyMethod | REQUIRED. Allowed values: `probabilistic`,`deterministic`,`eudx`,`fact`,`stt`,`null`,`ukf`,`spinglass`,`ens`,`other` |
```
Comment (arokem) [inline src/05-derivatives/05-diffusion-derivatives-tractography.md]: We need to add "pft" (particle filtering tractography, see [Girard et al., 2014](https://www.ncbi.nlm.nih.gov/pubmed/24816531))

File: src/05-derivatives/05-diffusion-derivatives-tractography.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+| Count              | REQUIRED. integer                                                                                                    |
```
Comment (arokem) [inline src/05-derivatives/05-diffusion-derivatives-tractography.md]: Is this the number of streamlines stored in the file?

**Comment by arokem** [comment]
Do you  want this merged ASAP, and come back to these comments later?

**Comment by oesteban** [comment]
> Do you want this merged ASAP, and come back to these comments later?

Yup, could you open a PR with your comments after this is merged?

**Comment by arokem** [comment]
Oh - no big green button! Looks like I don't have the permissions to merge this. :trombone:

**Comment by oesteban** [comment]
Fixed that!

**Comment by arokem** [comment]
:shipit:

**Pull request merged**